examples/va/nfv/hci-ovs-dpdk-sriov/edpm-pre-ceph/nodeset/values.yaml

```diff
@@ -0,0 +1,275 @@
+            mtu: 9000
+            use_dhcp: false
+            promisc: true
+          - type: sriov_pf
+            name: nic7
+            numvfs: 10
+            mtu: 9000
+            use_dhcp: false
+            promisc: true
+        # These vars are for the network config templates themselves and are
+        # considered EDPM network defaults.
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: nic1
+        # edpm_nodes_validation
+        edpm_nodes_validation_validate_controllers_icmp: false
+        edpm_nodes_validation_validate_gateway_icmp: false
+        dns_search_domains: []
+        gather_facts: false
+        # edpm firewall, change the allowed CIDR if needed
+        edpm_sshd_configure_firewall: true
+        edpm_sshd_allowed_ranges:
+          - 192.168.122.0/24
+        edpm_ceph_hci_pre_enabled_services:
+          - ceph_mon
+          - ceph_mgr
+          - ceph_osd
+          - ceph_rgw
+          - ceph_nfs
+          - ceph_rgw_frontend
+          - ceph_nfs_frontend
+        storage_mtu: 9000
+        storage_mgmt_mtu: 9000
+        storage_mgmt_vlan_id: 145
+        storage_mgmt_cidr: "24"
+        storage_mgmt_host_routes: []
+    networks:
+      - defaultRoute: true
+        name: ctlplane
+        subnetName: subnet1
+      - name: internalapi
+        subnetName: subnet1
+      - name: storage
+        subnetName: subnet1
+      - name: storagemgmt
+        subnetName: subnet1
+      - name: tenant
+        subnetName: subnet1
+    nodes:
+      edpm-compute-0:
+        ansible:
+          ansibleHost: 192.168.122.100
+        hostName: edpm-compute-0
+        networks:
+          - defaultRoute: true
+            fixedIP: 192.168.122.100
+            name: ctlplane
+            subnetName: subnet1
+          - name: internalapi
+            subnetName: subnet1
+          - name: storage
+            subnetName: subnet1
+          - name: storagemgmt
+            subnetName: subnet1
+          - name: tenant
+            subnetName: subnet1
+      edpm-compute-1:
+        ansible:
+          ansibleHost: 192.168.122.101
+        hostName: edpm-compute-1
+        networks:
+          - defaultRoute: true
+            fixedIP: 192.168.122.101
+            name: ctlplane
+            subnetName: subnet1
+          - name: internalapi
+            subnetName: subnet1
+          - name: storage
+            subnetName: subnet1
+          - name: storagemgmt
+            subnetName: subnet1
+          - name: tenant
+            subnetName: subnet1
+    services:
+      - bootstrap
+      - download-cache
+      - reboot-os
+      - configure-ovs-dpdk
+      - configure-network
+      - validate-network
+      - install-os
+      - ceph-hci-pre
+      - configure-os
+      - ssh-known-hosts
+      - run-os
+      - install-certs
+      - ovn
+      - neutron-ovn-igmp
+      - neutron-metadata
+      - neutron-sriov
+      - libvirt
+      - nova-custom-ovsdpdksriov
```

Comment:
we can move nova dataplane services during post ceph nodeset values right?
Looks like duplicated in pre and also in post.

Reply:
Deployment is done in the following way:
- preceph: ovs-dpdk-sriov working deployment  (as in ovs-dpdk-sriov architecture)
- postceph: Add hci to a ovs-dpdk-sriov deployment (as in hci architecture)
I would keep duplicated services for the following reason:
- This architecture is a mix of ovs-dpdk-sriov and hci architectures, so for maintenance reasons I would keep services as in those architectures.
- Preceph generates an working ovs-dpdk-sriov. That deployment is updated with ceph and some services are executed again in order to update it